### DISCUSSION TITLE
Additional Vehicle Types

### DISCUSSION THREAD

**Discussion by EvilBadger** [hargata/lubelog#459] [Ideas]
It would be great to have the ability to specify additional vehicle types I.e. motorcycles and bicycles. I have managed to somewhat use the existing vehicle types a more dedicated class would be a great addition for tracking maintenance on all vehicles.

**Comment by Jmanko16** [comment]
Has there been any update to this?  I am just starting to use lube logger, and it seems like it should be fine for the ATV and dirt bike, but wondering about bicycles etc.
